### DISCUSSION TITLE
auto-assign team to newly created customer

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#5059] [Questions & Answers]
Is there any chance that a setting is introduced by which newly created customers are auto-assign to a certain team (the work around with the sql command suggested in issue #1045 no longer works)?

**Comment by kevinpapst** [comment]
The teams from the current user should be automatically added AND you can select the team as well during creation.

But if you want a third setting: yes please, create a feature request if there is none yet.

**Reply by BeckeBauer** [reply]
#The teams from the current user should be automatically added

That is exactly what I am looking for. How can I activate this? 

I checked in the demo (logged in as susan_super and created a customer but it was not assigned to any of the 4 teams of susan_super)

**Reply by kevinpapst** [reply]
Ah, seems I was wrong, only for projects currently:

<img width="748" alt="Bildschirmfoto 2024-09-11 um 21 37 24" src="https://github.com/user-attachments/assets/042a2dbc-6359-45b8-964c-e3f44dd80a4b">

**Reply by BeckeBauer** [reply]
Any chance that this will be implemented on a customer level as well?
